packages/ethereum/constants.ts

```diff
@@ -11,25 +9,3 @@ export type Networks = 'hardhat' | 'localhost' | PublicNetworks
  */
 export type DeploymentTypes = 'testing' | 'development' | 'staging' | 'production'
 export type NetworkTag = DeploymentTypes | 'etherscan'
-
-export const networks: {
-  [network in PublicNetworks]: {
-    chainId: number
-    gas?: number
-  }
-} = {
-  xdai: {
-    chainId: 100,
-    gas: Number(utils.parseUnits('1', 'gwei'))
```

Comment:
Just noticed this line in the legacy implementation. It should be `gasPrice` instead of `gas`.